tests/std/tests/VSO_0000000_allocator_propagation/test.cpp

```diff
@@ -1046,27 +1024,27 @@ void test_vb() {
 
 using PCII = pair<const int, int>;
 
-template <typename K, typename V, typename C, typename A>
+template <class K, typename V, typename C, typename A>

```

Comment:
Nitpick: This file is being mostly converted to use `class` - the conversion is fine but the test code should consistently use one or the other.